inbm-lib/inbm_common_lib/constants.py

```diff
@@ -20,6 +20,8 @@
 RESPONSE_CHANNEL = 'manageability/response'
 EVENT_CHANNEL = 'manageability/event'
 TELEMETRY_CHANNEL = 'manageability/telemetry'
+# Used for Node updates to be sent to UDM
+UPDATE_CHANNEL = 'manageability/update'
```

Comment:
`NODE_UPDATE_CHANNEL` and `manageability/nodeUpdate` would be specific -- `update` sounds like it could apply to software updates